src/bus/webSocketConnection.js

```diff
@@ -9,10 +9,89 @@ class WebSocketConnection {
    * @param {string} organizationId UUID corresponding with an organization
    */
   constructor(webSocket, organizationId) {
+    this._jsonRpcId = 0;
     this._organizationId = organizationId;
     this._webSocket = webSocket;
   }
 
+  /**
+   * Sends a message to the message bus to authorize a channel
+   *
+   * @param {string} token JSON Web Signature containing the channel and actions needed for authorization
+   *
+   * @returns {Promise}
+   * @fulfill {Object} Whether you are authorized or not and any error
+   * @reject {errorEvent} The error event from the WebSocket
+   *
+   * @example
+   * axios.post(['https://', SERVICE_ADDRESS, '/authorizations/channels'].join(''), {
+   *   resources: [
+   *     {
+   *       resource: 'CHANNEL_YOU_NEED',
+   *       actions: ['publish', 'subscribe']
+   *     }
+   *   ]
+   * }, {
+   *   headers: {
+   *     Authorization: ['Bearer', ACCESS_TOKEN_FROM_PREVIOUS_STEP].join(' ')
+   *   }
+   * }).then((response) => {
+   *   const token = response.data;
+   *
+   *   contxtSdk.bus.connect('4f0e51c6-728b-4892-9863-6d002e61204d')
+   *     .then((webSocket) => {
+   *       webSocket.authorize(token).then((res) => {
+   *         if(res.error) {
+   *          console.log("Error: ", res.error)
+   *         }
+   *
+   *         console.log(res.authorized)
+   *       })
+   *       .catch((authErrorEvent) => {
+   *         console.log(authErrorEvent)
+   *       });
+   *     })
+   *     .catch((errorEvent) => {
+   *       console.log(errorEvent);
+   *     });
+   * });
```

Comment:
I think you can un-nest these `then`s a bit:


```
axios
  .post(
    ["https://", SERVICE_ADDRESS, "/authorizations/channels"].join(""),
    {
      resources: [
        {
          resource: "CHANNEL_YOU_NEED",
          actions: ["publish", "subscribe"]
        }
      ]
    },
    {
      headers: {
        Authorization: ["Bearer", ACCESS_TOKEN_FROM_PREVIOUS_STEP].join(" ")
      }
    }
  )
  .then(response => {
    const token = response.data;

    return contxtSdk.bus
      .connect("4f0e51c6-728b-4892-9863-6d002e61204d")
      .then((webSocket) => webSocket.authorize(token));
  })
  .then(response) => {
    if (response.error) {
      console.log("Error: ", response.error);
    }

    console.log(response.authorized);
  }
  .catch((error) => console.log("Error: ", error));
```

Also, have you thought about not including the info about how to go and "get" the token? It should be documented, but probably at a higher level -- it doesn't really have much to do with this method (outside of consuming the response, of course)